### PR TITLE
Fix outdated links to tiny-skia

### PR DESCRIPTION
**Pull request by dominikh**
This updates all links to tiny-skia from RazrFalcon to linebender. The changes to links to GitHub Pages are necessary, as the current links are 404. All other changes are optional and only avoid relying on redirections.

This PR is split into three commits:

- the necessary updates
- updates to redirecting links that aren't in the changelog
- updates to redirecting links that are in the changelog

I'm unsure about updating the links that redirect, and very unsure about changing the changelog. We can drop any of those two commits if we want to.

### CODE REVIEW

File: path/Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ edition = "2018"
 description = "A tiny-skia Bezier path implementation"
 documentation = "https://docs.rs/tiny-skia-path/"
 readme = "README.md"
-repository = "https://github.com/RazrFalcon/tiny-skia/tree/master/path"
+repository = "https://github.com/linebender/tiny-skia/tree/master/path"
```
Comment (DJMcNab) [inline path/Cargo.toml:9]: I think this should be:

```suggestion
repository = "https://github.com/linebender/tiny-skia"
```

The docs (https://doc.rust-lang.org/cargo/reference/manifest.html#the-repository-field) are a bit underspecified

**Review by DJMcNab — APPROVED** [review]
Thanks!

File: path/Cargo.toml (hunk 1 above)
Comment (dominikh) [reply path/Cargo.toml:9]: I'm not sure, either. But as is, the "Repository" link on https://crates.io/crates/tiny-skia-path is more useful IMO.

Comment (DJMcNab) [reply path/Cargo.toml:9]: Hmm, should this not at least have been changed to `/main/`?

Incidentally, the advantage of repository as documented is that it's the URL you'd use if you wanted to add a git dependency on the project.
A quick check:
- tracing uses the top-level repository
- serde uses the top-level repository
- Vello uses the top-level repository
- Naga uses the GitHub specific path

Comment (dominikh) [reply path/Cargo.toml:9]: > Hmm, should this not at least have been changed to `/main/`?

Probably, yeah. GitHub does redirect it, though, so it'll be just another cleanup.

I'll defer to actual Rust users on crate metadata, but I'm inclined to agree with you.

**Pull request merged**